test_data/rules/avoid_print.dart

```diff
@@ -4,10 +4,13 @@
 
 // test w/ `dart test -N avoid_print`
 
+import 'package:flutter/foundation.dart';
+
 void main() {
   print('ha'); // LINT
   [1,2,3].forEach(print); // LINT
   Future.value('hello').then(print); // LINT
+  if (kDebugMode) print(''); // OK
```

Comment:
Maybe a case w/ brace too?

```dart
if (kDebugMode) {
  print(''); // OK
}
```

Reply:
I'm guessing we don't care about cases like:

```dart
if (!kDebugMode) {
  /// ... 
} else {
  print(''); // OK
}
```

(To be clear, I don't think we should...)

Reply:
Done. And no, I wasn't planning on supporting any kind of flow control logic unless users really need it.